packages/docs/docs/search/graphql.mdx

```diff
@@ -222,9 +222,46 @@ Another common use is to filter an `extension` array by `url`:
 
 See the "[List Navigation](https://hl7.org/fhir/r4/graphql.html#list)" section of the FHIR GraphQL specification for more information.
 
+## Connection API
+
+Using the normal "List" search (i.e., "PatientList") is the most common way to search for resources. However, the FHIR GraphQL specification also supports the [Connection API](https://hl7.org/fhir/graphql.html#searching), which is a more complex way to search for resources.
+
+The most immediate advantage to the Connection API is better support for pagination and retrieving total counts. The Conenction API also includes more features from FHIR Bundle such as `mode` and `score`.
```

Comment:
> better support for pagination 
Would you mind adding a bit about this in your current example? I read the spec but was a bit confused about how to use it end-to-end.  A working example would be helpful

Reply:
Or remove "better support for pagination" until we have the cursor implemented